scoda/templates/scoda.html

```diff
@@ -10,6 +10,10 @@
     <meta name="description" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+     <!--Leaflet for maps-->
+     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.2/dist/leaflet.css"/>
```

Comment:
@riaansnyders could you please keep leaflet in the gulp js bundle. It helps us with load times.

Reply:
Completed. Will be in next push.